s2p/rasterization.py

```diff
@@ -123,34 +123,34 @@ def plyflatten_from_plyfiles_list(clouds_list, resolution, radius=0, roi=None, s
                         xsize, ysize,
                         radius, sigma)
 
-    utm_zone = utm_zone_from_ply(clouds_list[0])
-    utm_proj = geographiclib.utm_proj(utm_zone)
+    crs = crs_from_ply(clouds_list[0])
+    rasterio_crs = geographiclib.rasterio_crs(crs)
 
     # construct profile dict
     profile = dict()
     profile["tiled"] = True
     profile["compress"] = "deflate"
     profile["predictor"] = 2
     profile["nodata"] = float("nan")
-    profile["crs"] = utm_proj.srs
+    profile["crs"] = rasterio_crs
     profile["transform"] = affine.Affine(resolution, 0.0, xoff,
                                          0.0, -resolution, yoff)
 
     return raster, profile
 
 
-def utm_zone_from_ply(ply_path):
+def crs_from_ply(ply_path):
     _, comments = ply.read_3d_point_cloud_from_ply(ply_path)
-    regex = r"^projection: UTM (\d{1,2}[NS])"
-    utm_zone = None
+    regex = r"^projection: CRS (.*)"
+    crs = None
     for comment in comments:
         s = re.search(regex, comment)
         if s:
-            utm_zone = s.group(1)
+            crs = s.group(1)
 
-    if not utm_zone:
+    if not crs:
```

Comment:
Yes, it prevents rasterizing ply files that were computed by former versions of `s2p`.
We thought this was OK since this function was only used with ply files produced by the same version of `s2p`.
For "external" uses, we expected the users to rather use [`plyflatten`](https://github.com/cmla/plyflatten) which we will update to handle both old and new ply comments. (Actually this makes me realize that this external `plyflatten` is not yet used in `s2p`)

If you think that is a problem we can make sure to handle both cases here.

Reply:
Thanks, I agree that this should be ok as the function `plyflatten_from_plyfiles_list` is not likely to be used directly (and even less likely on old ply files) and "external" users should rather use [plyflatten](https://github.com/cmla/plyflatten).